app/client/components/holiday/holidayDateChooser.tsx

```diff
@@ -0,0 +1,396 @@
+                      <p>
+                        The first available date is{" "}
+                        <strong>
+                          {holidayStopsResponse.productSpecifics.firstAvailableDate.format(
+                            "dddd D MMMM"
+                          )}
+                        </strong>{" "}
+                        due to our printing and delivery schedule (notice
+                        period).
+                        <br />
+                        {creditExplainerSentence}
+                      </p>
+                      <div
+                        css={{
+                          fontFamily: sans,
+                          fontSize: "14px",
+                          marginBottom: "27px"
+                        }}
+                      >
+                        <InfoIcon />You can schedule one suspension at a time.
+                      </div>
+
+                      <DatePicker
+                        firstAvailableDate={
+                          holidayStopsResponse.productSpecifics
+                            .firstAvailableDate
+                        }
+                        issueDayOfWeek={
+                          holidayStopsResponse.productSpecifics.issueDayOfWeek
+                        }
+                        existingDates={holidayStopsResponse.existing.map(
+                          hsr => hsr.dateRange
+                        )}
+                        selectedRange={this.state.selectedRange}
+                        selectionInfo={this.getSelectionInfoElement(
+                          renewalDateMoment,
+                          combinedIssuesImpactedPerYear,
+                          holidayStopsResponse.productSpecifics.annualIssueLimit
+                        )}
+                        onSelect={this.onSelect(
+                          renewalDateMoment,
+                          combinedIssuesImpactedPerYear,
+                          holidayStopsResponse.productSpecifics
+                            .annualIssueLimit,
+                          productDetail.isTestUser
+                        )}
+                        dateToAsterisk={renewalDateMoment}
+                      />
+                      <div css={rightAlignedButtonsCss}>
+                        <HolidayQuestionsModal
+                          annualIssueLimit={
+                            holidayStopsResponse.productSpecifics
+                              .annualIssueLimit
+                          }
+                        />
+                        <Link css={cancelLinkCss} to=".." replace={true}>
+                          Cancel
+                        </Link>
+                        <div>
+                          <Button
+                            text="Review details"
+                            right
+                            disabled={
+                              !!this.state.validationErrorMessage ||
```

Comment:
not not?

Reply:
yes because disabled expects boolean, so if `this.state.validationErrorMessage` is `undefined` then type checker will complain. as with other example this is just cajoling truthy/falsy into boolean